apps/passport-client/new-components/screens/Home/NewHomeScreen.tsx

```diff
@@ -0,0 +1,456 @@
+    }
+  }, [currentPos, tickets]);
+
+  const renderedTickets = useMemo(() => {
+    ticketsRef.current = new Map();
+    return tickets.map(([eventName, eventTicketPack]) => {
+      return (
+        <TicketsContainer
+          key={eventTicketPack.map((pack) => pack.eventTicket.id).join("-")}
+        >
+          {eventTicketPack.map((pack) => {
+            return (
+              <CardBody
+                key={pack.eventId}
+                addOns={
+                  pack.addOns.length > 0
+                    ? {
+                        text: `View ${pack.addOns.length} add-on items`,
+                        onClick(): void {
+                          dispatch({
+                            type: "set-bottom-modal",
+                            modal: {
+                              addOns: pack.addOns,
+                              modalType: "ticket-add-ons"
+                            }
+                          });
+                        }
+                      }
+                    : undefined
+                }
+                ref={(ref) => {
+                  if (!ref) return;
+                  const group = ticketsRef.current.get(eventName);
+                  if (!group) {
+                    ticketsRef.current.set(eventName, [ref]);
+                    return;
+                  }
+                  group.push(ref);
+                }}
+                newUI={true}
+                pcd={pack.eventTicket}
+                isMainIdentity={false}
+              />
+            );
+          })}
+        </TicketsContainer>
+      );
+    });
+  }, [tickets, dispatch]);
+
+  if (!tickets.length)
+    return (
+      <AppContainer bg="gray">
+        <EmptyCard />
+        <FloatingMenu />
+        <NewModals />
+      </AppContainer>
+    );
+
+  return (
+    <AppContainer bg="gray" noPadding fullscreen>
+      <Container elWidth={eventCardWidth}>
+        <Scroller
+          gap={EVENT_GAP}
+          offset={(windowWidth - eventCardWidth) / 2}
+          ref={scrollRef}
+          scrollInPx={positionInPx(
+            currentPos,
+            eventCardWidth,
+            tickets.length - 1,
+            EVENT_GAP
+          )}
+          amount={tickets.length - 1}
+        >
+          {tickets.map(([eventName, packs], i) => {
+            const eventDetails = getEventDetails(packs[0]);
+            return (
+              <TicketCard
+                key={eventName}
+                address={eventName}
+                title={eventName}
+                ticketDate={new Date(
+                  eventDetails.timestampSigned
+                ).toDateString()}
+                imgSource={eventDetails.imageUrl}
+                ticketCount={packs.length}
+                cardColor={i % 2 === 0 ? "purple" : "orange"}
+              />
+            );
+          })}
+        </Scroller>
+        {SHOW_HELPER_LINES && <Line padding={(420 - eventCardWidth) / 2} />}
```

Comment:
where is this `420` coming from? could it be extracted into a constant with a comment?